internal/commands/storage/import.go

```diff
@@ -64,6 +64,8 @@ type importCommand struct {
 func (s *importCommand) InitCommand() {
 	flagSet := &pflag.FlagSet{}
 	flagSet.StringVar(&s.sourceLocation, "source-location", "", "Location of the source of the import. Can be a file or a URL.")
+	// TODO: is this flag actually required? Could we not just figure it out depending on the location?
+	// eg. if there's a file with the name given in location, use direct upload, otherwise validate url and use http import?
```

Comment:
Can we move this TODO & discussion to GitHub issues or somewhere else? 🤔  It will be lost & forgotten here in the code, hehe